src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin: auto;
+  width: 710px;
+  gap: $gap-between-days;
+
+  &__day {
+    cursor: pointer;
+    @include square($box-size);
+    background-color: $day-color;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 0.5s;
+
+    &:hover {
+      background-color: pink;
+      color: yellowgreen;
+      transform: translate(0, -20px);
+    }
+  }
+}
+
+$week-days:
+  "mon" 0,
+  "tues" $box-size + $gap-between-days,
+  "wed" $box-size * 2 + (2 * $gap-between-days),
+  "thurs" $box-size * 3 + (3 * $gap-between-days),
+  "fri" $box-size * 4 + (4 * $gap-between-days),
+  "sat" $box-size * 5 + (5 * $gap-between-days),
+  "sun" $box-size * 6 + (6 * $gap-between-days);
```

You can add variables in braces and then multiply sum by 2
```suggestion
  "wed" ($box-size  +  $gap-between-days) * 2,
  "thurs" $box-size * 3 + (3 * $gap-between-days),
  "fri" $box-size * 4 + (4 * $gap-between-days),
  "sat" $box-size * 5 + (5 * $gap-between-days),
  "sun" $box-size * 6 + (6 * $gap-between-days);
```

---

I think it looks better